diffkemp/llvm_ir/wrapper_build_finder.py

```diff
@@ -0,0 +1,68 @@
+"""
+LLVM source finder for projects that are entirely compiled in build phase
+of DiffKemp using the compiler wrapper.
+"""
+from diffkemp.llvm_ir.llvm_source_finder import LlvmSourceFinder
+import os
+import re
+
+
+class WrapperBuildFinder(LlvmSourceFinder):
+    """
+    LLVM source finder for projects that are entirely compiled in build phase
+    of DiffKemp using the compiler wrapper.
+    """
+    def __init__(self, source_dir, wrapper_db_filename):
+        LlvmSourceFinder.__init__(self, source_dir)
+        self.wrapper_db_filename = wrapper_db_filename
+        self.functions = {}
+        # Prepare for snapshot generation
+        self.initialize()
+
+    def str(self):
+        return "wrapper_build_tree"
+
+    def clone_to_dir(self, new_source_dir):
+        return WrapperBuildFinder(new_source_dir, self.wrapper_db_filename)
+
+    def initialize(self):
+        # Read LLVM IR file list from database generated by wrapper
+        # Note: only o: files are added, f: files contain code that is already
+        # present in the o: files
+        llvm_files = []
+        with open(self.wrapper_db_filename, 'r') as db_file:
+            for line in db_file.readlines():
+                line = line.rstrip()
+                if not line.startswith('o:'):
+                    continue
+                line = line.split(':')[1]
+                llvm_files.append(line)
```

Comment:
This feels more readable:
```suggestion
            llvm_files = [line.split(':')[1].rstrip()
                          for line in db_file.readlines()
                          if line.startswith(".o")]
```